src/components/TodoApp.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback, useState } from 'react';
+import { TodoList } from './TodoList';
+import { useTodoContext } from '../context/TodosContext';
+import { Filter } from './Filter';
+
+export const TodoApp: React.FC = () => {
+  const [query, setQuery] = useState<string>('');
+  const [activeCheckbox, secActiveCheckbox] = useState(false);
+  const { orderItems, setOrderItems } = useTodoContext();
+  const isNotCompletedTodos = orderItems.filter(item => !item.completed).length;
+  const isCompletedTodos = orderItems.filter(item => item.completed).length;
```

Comment:
```suggestion
  const incompoletedTodosCount = orderItems.filter(item => !item.completed).length;
  const completedTodosCount = orderItems.filter(item => item.completed).length;
```
These variables are not boolean